src/index.js

```diff
@@ -7,7 +7,9 @@ let initialized = false;
 let soundManager;
 if (typeof window !== 'undefined') {
   soundManager = require('soundmanager2').soundManager;
-  
+  if (process.env.NODE_ENV === 'production') {
```

Comment:
Can we remove this? This doesn't belong to this particular fix, and also I believe we can do better, there's a discussion for this in https://github.com/leoasis/react-sound/issues/13#issuecomment-263689258